Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -105,9 +107,10 @@ public struct DocumentID<Value: DocumentIDWrappable & Codable>:
   // MARK: - `Codable` implementation.
 
   public init(from decoder: Decoder) throws {
-    guard let reference = decoder.userInfo[documentRefUserInfoKey] as? DocumentReference else {
+    guard let reference = decoder
+      .userInfo[CodingUserInfoKey.documentRefUserInfoKey] as? DocumentReference else {
       throw FirestoreDecodingError.decodingIsNotSupported(
-        "Could not find DocumentReference for user info key: \(documentRefUserInfoKey)"
+        "DocumentID values can only be decoded with Firestore.Encoder"
```

Comment:
@andrewheard looking back at the implementation, this error message seems wrong. I'm ok with reverting this part of the change.

Reply:
Reverted.